src/test/resources/emails/sessionClosedEmailForInstructor.html

```diff
@@ -66,15 +66,18 @@
 </p>
 
 <p>
-  For any non-technical queries (e.g. assigned to the wrong team, misspelled name, request for deadline extension, etc.), you can contact your instructor(s) at: Instructor Not Yet Joined Course 1(instructorNotYetJoinedCourse1@email.tmt), Instructor1 Course1(instructor1@course1.tmt), Instructor3 Course1(instructor3@course1.tmt).
-</p>
-
-<p>
-  <strong>No action is required if you have already submitted.</strong>
-  <br><span style="background-color: #FFFF00">Any request for deadline extensions should be sent to the instructors of the course,</span> not to the TEAMMATES support team.
-</p>
-<p>
-  If you need any help or clarifications regarding TEAMMATES, please email us at ${support.email}.
+  <ul>
+    <li>
+      <strong>Requests for deadline extensions, problems regarding team/student data</strong> (e.g. wrong team, misspelled name),
+      and <strong>other non-technical queries</strong> about the feedback session:
+      <br>
+      Contact the instructors of the course: Instructor Not Yet Joined Course 1(instructorNotYetJoinedCourse1@email.tmt), Instructor1 Course1(instructor1@course1.tmt), Instructor3 Course1(instructor3@course1.tmt)
```

Comment:
Not your fault, but you would notice that between the instructor name and instructor email there is no whitespace (`name(email)` instead of `name (email)`). Let's identify the cause and fix this.

Reply:
Okay. Looking into this.

Reply:
@wkurniawan07 Found the cause. 

```diff
         for (InstructorAttributes coOwner : coOwners) {
             coOwnersEmailsLine
                     .append(coOwner.getName())
-                    .append('(')
+                    .append(" (")
                     .append(coOwner.getEmail())
                     .append("), ");

```
Just to clarify, as this is not related to this issue should I push the changes in this PR only or should I make another issue for this and send a PR to that issue?